### ISSUE TITLE
Uploading file does not work

### ISSUE THREAD

**Issue by jezekp** [dorianb96/EEG_WorkflowGUI#7]
If I want to upload a file only empty dialog appears and nothing is uploaded:

See:

![snimek obrazovky porizeny 2017-08-25 15-03-01](https://user-images.githubusercontent.com/1769981/29735361-433ea2b6-89ad-11e7-8fc7-8d58d20f9c45.png)

**Comment by dorianb96** [comment]
After too many hours of debugging deep HDFS calls, I managed not only to learn Hadoop internals but to find the root cause of why the copy operation does not work. 

Here are screenshots of the error that makes the process fail:

<img width="1440" alt="issue" src="https://user-images.githubusercontent.com/16664769/29753769-e7f29cb4-8b78-11e7-8cf8-607d1d8b4d7a.png">

<img width="1440" alt="issue1" src="https://user-images.githubusercontent.com/16664769/29753771-eaef80bc-8b78-11e7-83d4-cc3f5ba7cc22.png">

<img width="1440" alt="issue2" src="https://user-images.githubusercontent.com/16664769/29753772-ecf96634-8b78-11e7-9fc7-0ef04e09abad.png">


 
Basically it fails becuase of java.net.ConnectException when it switches from the usual 50070 webhdfs connection to secure webhdfs connection on port 1006. 
This is something I had never seen before. 

So a fix might be to open the port 1006 and that would allow for file transfers to data node. I would also open port 1004 just in case.

This is also a reference sheet to back that up: 
http://www.cloudera.com/documentation/manager/5-0-x/Cloudera-Manager-Installation-Guide/cm5ig_ports_cdh5.html
https://community.hortonworks.com/questions/27655/ports-used-for-distcp.html
